schemas/com.unbounce/event_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for Unbounce event context (https://documentation.unbounce.com/hc/en-us/articles/203510044-Using-a-Webhook#content6)",
+  "self": {
+    "vendor": "com.unbounce",
+    "name": "event_context",
```

Comment:
Why is this called an "event_context"? Is this Unbounce jargon? In Snowplow, an event and a context are different things...

Reply:
The original committer had two schemas for unbounce, an "event_context" and and "event_payload", but I do not know the reason he needed two (the data.json field can contain anything). I believe we can rename it to something more suitable (e.g. "form_post_event"). Any suggestions?

Reply:
> I do not know the reason he needed two

I have a theory, which is that the original idea was to turn the data.json field into the event (any event), and then attach the rest of the fields to this event as an "event context". This would explain the weird name.

This is really powerful - it would mean that you can configure your form against a JSON Schema that you configure yourself.

@knservis - was there any code in the adapter that supports this theory?